tests/channels/create_channel.ts

```diff
@@ -0,0 +1,26 @@
+import { cache, createChannel, delay } from "../../mod.ts";
+import { defaultTestOptions, tempData } from "../ws/start_bot.ts";
+import { assertEquals, assertExists } from "../deps.ts";
+
+Deno.test({
+  name: "[channel] create a new channel",
+  async fn() {
+    const channel = await createChannel(tempData.guildId, {
+      name: "Discordeno-test",
+    });
+
+    // Assertions
+    assertExists(channel);
+
+    tempData.channelId = channel.id;
+    assertEquals(tempData.channelId, channel.id);
```

Comment:
Why do you have this check?

Reply:
When i was first testing, it wasn't working properly. IDK why but guildId remained empty and so i added that to alert just in case. We can probably remove these in the future but i think for the time being its safe and basically no impact to performance to have this sanity check.